invoices/invoiceregistry_test.go

```diff
@@ -125,12 +126,20 @@ func TestInvoiceRegistry(t *testing.T) {
 		pgFixture.TearDown(t)
 	})
 
+	// sqliteConstructorMu is used to ensure that only one thread can call
+	// the sqldb.NewTestSqliteDB constructor at a time. This is a temporary
+	// workaround that can be removed once this race condition in the sqlite
+	// repo is resolved: https://gitlab.com/cznic/sqlite/-/issues/180
+	var sqliteConstructorMu sync.Mutex
```

Comment:
Nice! I think unfortunately we'll need to same treatment for `invoices_test.go` too. Requires that we move the mutex to the global scope though but i think it's ok given it's test only.

Reply:
oh yeah ofc, good catch 🙈  updated!

Reply:
Nice, thanks! :)